spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/TestSparkDistributionAndOrderingUtil.java

```diff
@@ -1441,6 +1441,132 @@ public void testRangePositionDeltaDeletePartitionedTable() {
         table, DELETE, expectedDistribution, SPEC_ID_PARTITION_FILE_POSITION_ORDERING);
   }
 
+  // ===================================================================================
+  // Distribution and ordering for merge-on-read UPDATE operations with position deletes
+  // ===================================================================================
+  //
+  // update mode is NOT SET -> CLUSTER BY _spec_id, _partition + LOCALLY ORDER BY _spec_id, _partition, _file, _pos
```

Comment:
Same like in merge-on-read DELETE. Here we are dealing with full rows so it does not seem like a good idea to group everything into a single task for unpartitioned tables. Ideas are welcome.

Reply:
I think this is a reasonable default. We don't expect high volume for merge-on-read, nor do we expect unpartitioned tables to be high volume. If they are, then setting an explicit mode=none will fix it.

Reply:
Okay, let's keep it for now then.